test-tools/wamr-ide/VSCode-Extension/src/taskProvider.ts

```diff
@@ -40,7 +45,11 @@ export class WasmTaskProvider implements vscode.TaskProvider {
                             : (this._script.get('buildScript') as string),
                     options: {
                         executable: this._script.get('buildScript'),
-                        shellArgs: [targetName, os.platform()],
+                        shellArgs: [
+                            targetName,
+                            this._wamrVersion,
+                            os.platform(),
```

Comment:
maybe the `os.platform()` can be removed since it is not used by `build scripts (build.bat & build.sh)`

Reply:
Should be also OK to reserve the argument, maybe we need to use it in the future.